code/game/gamemodes/dynamic/dynamic_rulesets.dm

```diff
@@ -18,12 +18,10 @@
 	var/list/mob/candidates = list()
 	/// List of players that were selected for this rule
 	var/list/datum/mind/assigned = list()
-	/// Preferences flag such as ROLE_WIZARD that need to be turned on for players to be antag
-	var/antag_flag = null
+	/// The /datum/role_preference typepath used for this ruleset.
+	var/role_preference = null
 	/// The antagonist datum that is assigned to the mobs mind on ruleset execution.
 	var/datum/antagonist/antag_datum = null
-	/// The required minimum account age for this ruleset.
-	var/minimum_required_age = 7
```

Comment:
There are no hours checks currently no, I felt it a bit out of scope since this PR was already massive and it might be a lot more work. Now I'll be tying it to antag datums instead but it's not something I want to do in this PR necessarily if I don't have it.

Not that account age checks were really doing anything other than for the newest of the new.

Reply:
Added living hours checks for basic dynamic rulesets. Gamemodes not right now